Tests/InitResourceDbSubscriber.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests;
+
+class InitResourceDbSubscriber implements \Enlight\Event\SubscriberInterface
```

Comment:
what kind of magic is this?? :D
please add a small comment why this is necessary and what will be broken if it is removed